### PR TITLE
Login endpoint

### PR DESCRIPTION
**Pull request by DominykasSip**
We should somehow encrypt credentials in the frontend and decrypt in the backend because now the credentials can be seen  in plain text when you use the login endpoint.

### CODE REVIEW

**Review by DominykasSip — COMMENTED** [review]
I don't know if this fixes the problem but at least we don't send credentials through the URL

**Review by DonatasPuc — APPROVED** [review]
I think it's better. Should be ok for now.

**Comment by JesusGonzalezA** [comment]
> We should somehow encrypt credentials in the frontend and decrypt in the backend because now the credentials can be seen in plain text when you use the login endpoint.

All the data will be automatically encrypted when using a secure protocol, such as https.

File: backend/WebApi/Controllers/RestaurantController.cs [hunk 1]
```diff
@@ -186,5 +186,17 @@ public IActionResult GetAllFoodFromRestaurant(string id, string sortOrder = null
...
+        /// <summary>
+        /// Shows if the log in credentials are correct
+        /// </summary>
+        /// <param name="creds">Credentials of the restaurant</param>
+        /// <returns></returns>
+        [HttpPost("Login", Name = nameof(Login))]
+        [ProducesResponseType((int)HttpStatusCode.OK)]
+        public IActionResult Login([FromBody] Credentials creds)
+        {
+            return Ok(_restaurantService.Login(creds));
```
Comment (JesusGonzalezA) [inline backend/WebApi/Controllers/RestaurantController.cs:199]: Can you send instead the id of the restaurant as we do in the registration endpoint? Without the id we don't know who logged in.

**Review by JesusGonzalezA — CHANGES_REQUESTED** [review]

File: backend/WebApi/Controllers/RestaurantController.cs (hunk 1 above)
Comment (DominykasSip) [reply backend/WebApi/Controllers/RestaurantController.cs]: Changed it

**Review by audrius-savickas — APPROVED** [review]

**Pull request merged**